Source/Blazorise/wwwroot/fileEdit.js

```diff
@@ -17,28 +17,43 @@ export function initialize(adapter, element, elementId) {
         // Reduce to purely serializable data, plus build an index by ID
         element._blazorFilesById = {};
 
-        var fileList = Array.prototype.map.call(element.files, function (file) {
-            var fileEntry = {
-                id: ++nextFileId,
-                lastModified: new Date(file.lastModified).toISOString(),
-                name: file.name,
-                size: file.size,
-                type: file.type
-            };
-            element._blazorFilesById[fileEntry.id] = fileEntry;
-
-            // Attach the blob data itself as a non-enumerable property so it doesn't appear in the JSON
-            Object.defineProperty(fileEntry, 'blob', { value: file });
-
-            return fileEntry;
-        });
+        var fileList = mapElementFilesToFileEntries(element);
 
         adapter.invokeMethodAsync('NotifyChange', fileList).then(null, function (err) {
             throw new Error(err);
         });
     });
 }
 
+function mapElementFilesToFileEntries(element) {
```

Comment:
![image](https://user-images.githubusercontent.com/900302/165107917-a8000e12-b157-4b02-88e2-0d0fc76f28d9.png)

Reply:
(untested code haha) was done before leaving home that one. Do you know what extension is validating that for you? Or is it VS?

Reply:
This is just devtools in chrome.

Reply:
Ahh I tought vs was being smart and telling you beforehand. :)